### DISCUSSION TITLE
HashDB import tool with NSRL CAID

### DISCUSSION THREAD

**Discussion by vtgdias** [sepinf-inc/IPED#1210] [Q&A] [answered]
I am trying to import NSRL CAID, following directions from https://github.com/sepinf-inc/IPED/wiki/User-Manual#hashes-database-version--400. 

I get the following error:

```
Reading PROJECT_VIC file X:\NSRL-CAID-JSONs\NSRL-CAID-3GP.json...
Error reading file: X:\NSRL-CAID-JSONs\NSRL-CAID-3GP.json
java.lang.RuntimeException: Error: Unknown ProjectVic JSON data model!
        at iped.engine.hashdb.HashDBTool.readProjectVIC(HashDBTool.java:820)
        at iped.engine.hashdb.HashDBTool.readFile(HashDBTool.java:556)
        at iped.engine.hashdb.HashDBTool.readFiles(HashDBTool.java:546)
        at iped.engine.hashdb.HashDBTool.run(HashDBTool.java:131)
        at iped.engine.hashdb.HashDBTool.main(HashDBTool.java:113)
```

Downloaded latest version of CAID, thta looks like this:

```
{
  "odata.metadata":"http://github.com/ICMEC/ProjectVic/DataModels/1.2.xml#Media","value":[
    {
      "odata.id":"Media(\"6f55e592692e6b549fbe316ccf25a5fc\")","MediaFiles@odata.navigationLinkUrl":"/Media(6f55e592692e6b549fbe316ccf25a5fc)/MediaFiles","MediaFiles":[
      {
        "Media@odata.navigationLinkUrl":"/MediaFiles(FileName='lalinea3.3gp',FilePath='res/raw',MD5=6f55e592692e6b549fbe316ccf25a5fc)","MD5":"6f55e592692e6b549fbe316ccf25a5fc","FileName":"lalinea3.3gp","FilePath":"res/raw"
      }
      ],"MediaID":0,"MD5":"6f55e592692e6b549fbe316ccf25a5fc","SHA1":"6f51f7e147ef1988679f7b07c9bbdb4f1eaf580c","Category":8,"MediaSize":"2234416"
    },
```

Other thing, is it possible to download newer version of LED? or at least the hashes in csv format to be imported with hashdb?

**Comment by lfcnassif** [comment, marked as answer]
Support for NSRL CAID was integrated recently (#1177) and not release yet, it will be soon. The manual was updated early, sorry about the confusion.

> Other thing, is it possible to download newer version of LED? or at least the hashes in csv format to be imported with hashdb?

Well, this project is not in position to distribute them... For newer LED version, you should contact its author @tc-wleite. For access to Brazilian Federal Police hash sets in CSV (or other) format AND if you are law enforcement, you should ask them officially to sepinf.inc.ditec@pf.gov.br, so further instructions will be provided privately.

**Comment by vtgdias** [comment]
Tks @lfcnassif, I am from forensic lab in Rio (PCERJ), i ll send them an email! As for latest LEDs version, do you have @tc-wleite contacts?

**Reply by lfcnassif** [reply]
Well, he watches this repo, so I think he will answer you when he could/have time :-)

**Reply by wladimirleite** [reply]
@vtgdias, I am on vacation. I will contact you privately when I return, in two weeks.

**Reply by wladimirleite** [reply]
@vtgdias, I couldn't find your contact. Can you send me an email about this? wladimir.wlcl@pf.gov.br
